functions/tsconfig.json

```diff
@@ -6,6 +6,9 @@
     "outDir": "lib",
     "sourceMap": true,
     "strict": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
```

Comment:
Are these options still needed or just part of debugging?

Reply:
As the index.ts file was throwing error for adding json file. I added these options. They are needed.

Reply:
ok 👍